src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/ArtworksRail.tsx

```diff
@@ -39,48 +41,84 @@ export const ArtworksRail: FC<ArtworksRailProps> = props => {
   const { state } = props
   const [artworks, setArtworks] = useState<DiscoveryArtwork[]>([])
   const [isLoading, setIsLoading] = useState<boolean>(true)
+  const [excludeArtworkIds, setExcludeArtworkIds] = useState<string[]>([])
```

Comment:
I decided to store these client side and send with the `fetchArtworks` request (L49) for a few reasons:

1. This prevents us from having needing the weaviate user client side, as we aren't reliant on the `likedArtworks` or `dislikedArtworks` fields. 
2. This allows us to filter out `artworks` the user has liked or disliked from the rail (note: a refresh would clear the filter list).
3. We need a state change in the `ArtworksRails` component  to trigger a refetch of the artworks.